tests/components/broadlink/__init__.py

```diff
@@ -76,6 +76,21 @@ def __init__(
         self.timeout: int = timeout
         self.fwversion: int = fwversion
 
+    async def setup(self, hass, mock_api=None, mock_entry=None):
```

Comment:
maybe we rename this to setup_config_entry or setup_entry?

Reply:
Sure. It's better now. Thank you.